res/openvpn-gui-res-no.rc

```diff
@@ -184,12 +184,12 @@ FONT 8, "Microsoft Sans Serif"
 LANGUAGE LANG_NORWEGIAN, SUBLANG_DEFAULT
 BEGIN
     ICON ID_ICO_APP, 0, 8, 16, 21, 20
-    LTEXT "OpenVPN GUI v" PACKAGE_VERSION " - Ett Windows-Grensesnitt mot OpenVPN\n" \
+    LTEXT "OpenVPN GUI v" PACKAGE_VERSION " - Et Windows-grensesnitt mot OpenVPN\n" \
           "Copyright (C) 2004-2005 Mathias Sundman <info@openvpn.se>\n" \
           "http://openvpn.se/", 0, 36, 15, 206, 26
-    LTEXT "OpenVPN - En applikasjon for sikker overføring av IP nett " \
-          "over en enkel UDP port, med støtte for SSL/TLS-basert " \
-          "sessjons autentisering og nøkkel håndtering, pakkekryptering, " \
+    LTEXT "En applikasjon for sikker kommunikasjon mellom nettverk over en usikker " \
+          "kommunikasjonskanal på én enkelt port, med støtte for SSL/TLS-basert " \
```

Comment:
Removal of the word "OpenVPN" here is by mistake, right? We are referring to two applications in these paragraphs: OpenVPN-GUI in first and OpenVPN  in second.